api/helpers/uploadedFileErrorChecks.js

```diff
@@ -133,33 +185,32 @@ function getStorageNameFromSession(file, req) {
     return fileWithStorageNameFound.storageName;
 }
 
-function addUnsubmittedTag(file, req) {
-    const fileStorageName = getStorageNameFromSession(file, req);
-    const fileBelongsToUnsubmittedApplication = {
-        Key: 'app_status',
-        Value: 'UNSUBMITTED',
-    };
-
-    const params = {
-        Bucket: req._sails.config.upload.s3_bucket,
-        Key: fileStorageName,
-        Tagging: {
-            TagSet: [fileBelongsToUnsubmittedApplication],
-        },
-    };
-
-    s3.putObjectTagging(params, (err) => {
-        if (err) {
-            throw new Error(err);
-        }
-    });
-    sails.log.info(`Only UNSUBMITTED tag added to ${fileStorageName}`);
+async function addUnsubmittedTag(file, req) {
+    try {
+        const fileStorageName = getStorageNameFromSession(file, req);
+        const fileBelongsToUnsubmittedApplication = {
+            Key: 'app_status',
+            Value: 'UNSUBMITTED',
+        };
+        const params = {
+            Bucket: req._sails.config.upload.s3_bucket,
+            Key: fileStorageName,
+            Tagging: {
+                TagSet: [fileBelongsToUnsubmittedApplication],
+            },
+        };
+
+        await s3.send(new PutObjectTaggingCommand(params));
+        sails.log.info(`Only UNSUBMITTED tag added to ${fileStorageName}`);
+    } catch (err) {
+        throw new Error(`addUnsubmittedTag ${err}`);
+    }
 }
 
 function scanResponses(scanResults, file, req = null, forS3 = false) {
     const { isInfected, viruses } = scanResults;
+    console.log('gets here 3', scanResults.viruses);
```

Comment:
do you mean to console.log here?

Reply:
Ah no this was for unit testing, I'll get rid of it :)